iife-wrapper.js

```diff
@@ -4,29 +4,24 @@ var Pretender = (function(self) {
   }
 
   var appearsBrowserified =
-    typeof self !== 'undefined' &&
-    typeof process !== 'undefined' &&
-    (Object.prototype.toString.call(process) === '[object Object]' ||
-      Object.prototype.toString.call(process) === '[object process]');
+    typeof self !== "undefined" &&
```

Comment:
Ugh seems missing local prettier config, maybe adding one in another PR and keep this one clean.

Reply:
👍

Reply:
https://github.com/pretenderjs/pretender/pull/298